docs/upgrade/UPGRADE-v7.1.0.md

```diff
@@ -0,0 +1,173 @@
+# [Upgrade from v7.0.0 to v7.1.0]
+
+This guide contains instructions to upgrade from version v7.0.0 to v7.1.0.
+
+**Before you start, don't forget to take a look at [general instructions](/UPGRADE.md) about upgrading.**
+There you can find links to upgrade notes for other versions too.
+
+## [shopsys/framework]
+### Application
+- *(low priority)* to add support of functional tests of Redis ([#846](https://github.com/shopsys/shopsys/pull/846))
+    - download [`snc_redis.yml`](https://github.com/shopsys/project-base/blob/master/app/config/packages/test/snc_redis.yml) to `app/config/packages/test/snc_redis.yml`
+    - download [`RedisFacadeTest.php`](https://github.com/shopsys/project-base/tree/master/tests/ShopBundle/Functional/Component/Redis/RedisFacadeTest.php) to your tests directory `tests/ShopBundle/Functional/Component/Redis/`
+- *(low-priority)* by changes realized in [#808 redesign print page of product detail page](https://github.com/shopsys/shopsys/pull/808) was created new folder `print` in `src/Shopsys/ShopBundle/Resources/styles/front/common`. In the pull request were realized changes for styling print page of initial Shopsys Framework configuration. In order to apply changes in your project there is need to do following steps.
```

Comment:
This really bothers me - "realized" is used wrongly three times here, "performed" or "implemented" should be used instead, could you please fix it?

Reply:
updated